### ISSUE TITLE
Single page pdf appearance

### ISSUE THREAD

**Issue by bmblb** [bryntum/support#121]
Problem:
when you export 1000 rows in grid to a single page you get a pdf with 1 page of requested size (A4 by default) and all content squeezed into it. Look like this: 
![image](https://user-images.githubusercontent.com/45400762/71255731-cef07b80-233f-11ea-94e5-71eaafaf0e6b.png)

Old version used to ignore page size, there was one very long page, but content looked readable. But when printed it would look the same.

In other words, new export shows how would result look printed, old export showed unscaled content inside the pdf.

Question is: what should be the result of exporting too much content to single page?
We could maybe add a flag to switch scaling on/off.
